src/index.js

```diff
@@ -1,51 +1,28 @@
 import './style.css';
+import {
+  newTaskInput, addTask, removeTask, renderList, editTask, updateTask,
+} from './modules/utilities.js';
 
-const tasks = [
-  {
-    description: 'I am a task',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'I am a task',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'I am a task',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'I am a task',
-    completed: false,
-    index: 3,
-  },
-  {
-    description: 'I am a task',
-    completed: false,
-    index: 4,
-  },
-];
-const tasksContainer = document.querySelector('ul');
-
-const renderList = () => {
-  tasks.forEach((task) => {
-    const li = document.createElement('li');
-    const checkbox = document.createElement('input');
-    const p = document.createElement('p');
-    const span = document.createElement('span');
-    const icon = document.createElement('i');
-
-    checkbox.setAttribute('type', 'checkbox');
-    p.textContent = `${task.description}`;
-    icon.classList.add('las');
-    icon.classList.add('la-ellipsis-v');
-
-    span.appendChild(icon);
-    li.append(checkbox, p, span);
-    tasksContainer.appendChild(li);
+renderList();
+newTaskInput.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter') {
+    addTask();
+  }
+});
+document.querySelectorAll('.la-ellipsis-v').forEach((btn) => {
+  btn.addEventListener('click', (e) => {
+    editTask(e.target.dataset.id);
   });
-};
-
-renderList();
\ No newline at end of file
+});
+document.querySelectorAll('.edit-task').forEach((task) => {
+  task.addEventListener('keypress', (e) => {
+    if (e.key === 'Enter') {
+      updateTask(e.target.dataset.id);
+    }
+  });
+});
+document.querySelectorAll('.la-trash').forEach((btn) => {
+  btn.addEventListener('click', (e) => {
+    removeTask(e.target.dataset.id);
+  });
+});
```

Comment:
https://user-images.githubusercontent.com/65068771/220793851-c6a4e9c1-de46-4030-9b93-374d36e53a35.mov


- [ ] Kindly make sure you can edit or delete a task after adding it currently the functionality is just working when the page is refreshed. Kindly work on that.